include/subgroup/tile/impl/load_xe.hpp

```diff
@@ -79,7 +79,7 @@ template <
     typename payload_t>
 __XETLA_API typename std::enable_if_t<
     detail::check_load_type<tile_t, payload_t>::is_global_block_2d &&
-    arch_has_2d_load_store(payload_t::arch_tag)>
+    payload_t::arch_tag == gpu_arch::XeHpc>
```

Comment:
I think arch_has_2d_load_store(payload_t::arch_tag) is better than a fixed tag condition

Reply:
rebase error`  fixed`~